litmus-portal/frontend/src/components/WorkflowCard/CardContent.tsx

```diff
@@ -87,6 +87,7 @@ const CardContent: React.FC<preDefinedWorkflowData> = ({
           ) : (
             <span />
           )}
+          {description ? description.length < 28 ? <br /> : <div /> : <span />}
```

Comment:
What's the use of 28 value here?

Reply:
This is required to make the predefined workflow card sizes consistent, it's a necessary fix for the UI. @chandankumar4

Reply:
This is a very bad fix...with absolute numbers in the jsx

Reply:
If the description is too long it would automatically fit into the provided div with which could be of a customizable width. If you want to shorten up the text to fit the display you can use `string.substring(0, givenLengthToShorten) + "...";` or `lodash` to truncate

Reply:
Yeah for now we have to ahead with this temporary fix, if we wan’t a proper UI. I guess team 3 can make the changes in the design of the cards to fix the UI bug which i fixed temporarily. cc: @chandankumar4